### PR TITLE
Add procfile cloudinary

### PR DESCRIPTION
**Pull request by tuoanhnt95**
Added cloudinary gemfile, change dev and prod. environment from local to cloudinary, added cloudinary key to .env file

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@
   end
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  config.active_storage.service = :cloudinary
```
Comment (mengrui-song) [inline config/environments/development.rb:38]: for our seed I want to use local directory to save the result PDF, I don't know how to use the active_storage as local and cloudinary at the same time.

File: config/environments/production.rb [hunk 2]
```diff
@@ -39,7 +39,7 @@
   # config.action_dispatch.x_sendfile_header = "X-Accel-Redirect" # for NGINX
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  config.active_storage.service = :cloudinary
```
Comment (mengrui-song) [inline config/environments/production.rb:42]: same question

File: config/environments/development.rb (hunk 1 above)
Comment (tuoanhnt95) [reply config/environments/development.rb:38]: According to Trouni, pdf can be stored with Cloudinary.

File: config/environments/production.rb (hunk 2 above)
Comment (tuoanhnt95) [reply config/environments/production.rb:42]: see above

**Pull request merged**